test/playwright/utils/navigation.ts

```diff
@@ -319,9 +319,22 @@ export const dismissTranslationBanner = async (page: Page) => {
 export const selectHomepageSearchType = async (
   page: Page,
   searchType: SupportedSearchType,
-  dir: LanguageDirection = "ltr"
+  dir: LanguageDirection = "ltr",
+  headerMode: HeaderMode = NEW_HEADER
 ) => {
   const pageWidth = page.viewportSize()?.width
+  // TODO: replace when the search types can only be selected using the popover/modal
+  console.log("Selecting homepage search type in header mode", headerMode)
```

Comment:
```suggestion
  // console.log("Selecting homepage search type in header mode", headerMode)
```

Any reason to keep this?

Reply:
It's removed back in the further PRs :)